docs/usage/edsl.rst

```diff
@@ -1,20 +1,21 @@
-Tile eDSL 
+Tile EDSL 
```

Comment:
Can we use `eDSL` instead?

Reply:
A few members of the team expressed that `eDSL` was originally a typo and `EDSL` is the correct capitalization. That is why Sana capitalized them here. We should probably try and correct the understanding across the team if `eDSL` is the preferred capitalization.

I don't have a strong opinion either way, but can we at least be consistent across our documentation?

Reply:
Let's see if we can come to a consensus tomorrow when I'm in. In the meantime, I'd like to keep this particular PR clean from any changes in the copy (e.g. text).

Reply:
SGTM